logger.go

```
@@ -79,11 +82,50 @@ func setupTargetLogger(flags commandLineFlags, logTarget, commandOutput string)
+	err := w.LogCloser.Close()
+	if err != nil {
+		return err
+	}
+	// Open logfile for reading
+	logData, err := os.Open(w.logfilePath)
+	if err != nil {
+		return err
+	}
+	// Upload logfile to server
+	resp, err := http.Post(w.logUploadTarget, "application/octet-stream", logData)
+	defer resp.Body.Close()
```

golangci-lint (1.64.8)

httpresponse: using resp before checking for errors

(govet)